### ISSUE TITLE
Brand Assets Guidelines Discussion

### ISSUE THREAD

**Issue by Suzie97** [vala-lang/brand#1]
# Vala Brand Guidelines

### "Vala" Logotype

![Logo Usage - Colour](https://user-images.githubusercontent.com/68198116/206915528-8dee3906-5474-453f-be64-ecb03b15a7e1.png)

- This is the primary way to refer to Vala.
- Use the colour code **#A56DE2** when the background is close to black and the colour code **#7239B3** when the background is close to white.

![WCAG](https://user-images.githubusercontent.com/68198116/206915543-76715c4a-24d0-4f90-ab9a-3cef2bb60a4b.png)

- For other background colours, it is recommended to use the colours from the brand palette and maintain at least a WCAG AA level contrast.

![Logo Usage](https://user-images.githubusercontent.com/68198116/206915551-a13ff54c-0dae-4b6c-ad41-911ae67b1f83.png)

### "V" Logomark

![Logo](https://user-images.githubusercontent.com/68198116/206915559-eec20c3a-d2d3-406d-acad-efeb829d4198.png)

- The "V" logomark should be used when less real estate is available, especially when a square ratio is required or when primary "Vala" logotype feels too cramped.

#### Usage Example
<img width="723" alt="Logomark Usage" src="https://user-images.githubusercontent.com/68198116/206915592-3a702b56-dfc6-4e82-9e51-6544d712b1a2.png">

### Vala Purple

![Vala Colour](https://user-images.githubusercontent.com/68198116/206915572-5f92f3b1-f5ce-40ff-9e79-626cc38abc4d.png)

- Use it for all hero statements

### Vala Gradient

![Vala Gradient](https://user-images.githubusercontent.com/68198116/206915578-bfaa675e-1aeb-44eb-8134-be22847a3ce3.png)

- Use it as needed

### Colour Palette
![Vala Palette](https://user-images.githubusercontent.com/68198116/206915682-370a3fb5-cd76-47c6-abb9-8f071c2a4fbf.png)

**Comment by Suzie97** [comment]
These are the assets and initial guidelines I made. We can discuss any changes in this issue and then upload these assets to the repo and also make a dedicated brand page on the website.

**Comment by pervoj** [comment]
It would be great to add typography guidelines too

**Comment by Suzie97** [comment]
> It would be great to add typography guidelines too

Do we have any official fonts that we use for Vala related projects?

**Comment by pervoj** [comment]
> > It would be great to add typography guidelines too
> 
> Do we have any official fonts that we use for Vala related projects?

I don't think so. Valadoc uses mostly Open Sans, Vala website Inter, and that's all.

**Comment by colinkiama** [comment]
> > It would be great to add typography guidelines too
> 
> Do we have any official fonts that we use for Vala related projects?

Also, the logo's font is Lobster and GNOME uses Cantarell.

**Comment by Suzie97** [comment]
> > > It would be great to add typography guidelines too
> > 
> > 
> > Do we have any official fonts that we use for Vala related projects?
> 
> Also, the logo's font is Lobster and GNOME uses Cantarell.

I think the fact that the Lobster font is used for the logo does not need to be explicitly mentioned. We don't use it anywhere else (not even in the website).

And I think we need to figure out whether or not to use Inter or Cantarell or both as the official font in its own separate issue, since that will need a deeper discussion because I've seen some inconsistent of these fonts in official GNOME sites as well.

I think for the first version of the guidelines, the current agenda is enough. We just need to finalise them and figure out a way to present them in the website.

Let me know what you think.

**Comment by pervoj** [comment]
> > > > It would be great to add typography guidelines too
> > > 
> > > 
> > > Do we have any official fonts that we use for Vala related projects?
> > 
> > 
> > Also, the logo's font is Lobster and GNOME uses Cantarell.
> 
> I think the fact that the Lobster font is used for the logo does not need to be explicitly mentioned. We don't use it anywhere else (not even in the website).
> 
> And I think we need to figure out whether or not to use Inter or Cantarell or both as the official font in its own separate issue, since that will need a deeper discussion because I've seen some inconsistent of these fonts in official GNOME sites as well.
> 
> I think for the first version of the guidelines, the current agenda is enough. We just need to finalise them and figure out a way to present them in the website.
> 
> Let me know what you think.

GNOME has it's own https://brand.gnome.org/ where the fonts usage is explained. Cantarell is used in the DE and apps and on websites for headings. Inter just on websites for body texts.
